js/page.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+    const iframe = document.querySelector('.s-page-summary iframe');
+    if (!iframe) return; // no iframe detected
+
+    const pageTitle = document.querySelector('.s-page-title');
+    const link = createElement("a", [], {
```

Comment:
Per our discussion on Discord, we'll probably take another pass at the styling here. My first intuition is that we probably won't make the entire title a link, just the button; we'll probably also change the icon. That said, nice work!